src/benchmarks/micro/libraries/System.Memory/Span.cs

```diff
@@ -71,12 +77,18 @@ public void SetupIndexOf()
         [Benchmark]
         public int IndexOfAnyTwoValues() => new System.Span<T>(_emptyWithSingleValue).IndexOfAny(_notDefaultValue, _notDefaultValue);
 
+        [MemoryRandomization(true)]
+        [IterationCount(50)]
+        [Outliers(Perfolizer.Mathematics.OutlierDetection.OutlierMode.DontRemove)]
         [Benchmark]
         public int IndexOfAnyThreeValues() => new System.Span<T>(_emptyWithSingleValue).IndexOfAny(_notDefaultValue, _notDefaultValue, _notDefaultValue);
 
         [Benchmark]
         public int IndexOfAnyFourValues() => new System.Span<T>(_emptyWithSingleValue).IndexOfAny(new ReadOnlySpan<T>(_fourValues));
```

Comment:
Is there a specific reason not to do the randomization for both?

Reply:
I just went with the initial set that Adam had laid out. We might expand these going forward however.